### PR TITLE
Improve persistence transformers

### PR DESCRIPTION
**Pull request by neykov**
* Accept a list of single key-value maps so one file can contain multiple actions with same name
* When renaming a class replace the new name in the class tag, type element, class attribute

### CODE REVIEW

**Comment by aledsage** [comment]
Test failure looks unrelated, and has been happening far too often. Perhaps we need to mark everything that requires an open port to be integration tests?!

    java.lang.RuntimeException: unable to find a free port at or above 8080
    	at brooklyn.util.net.Networking.nextAvailablePort(Networking.java:156)
    	at io.brooklyn.camp.CampServer$CampServerUtils.startServer(CampServer.java:151)

File: core/src/main/java/brooklyn/entity/rebind/transformer/CompoundTransformer.java [hunk 1]
```diff
@@ -181,6 +188,18 @@ public Builder changeCatalogItemId(String oldSymbolicName, String newSymbolicNam
...
+        /**
+         * Updates all references to a class to a new value
+         * @param oldName the old name of the class
+         * @param newName the new name of the class to be used instead
+         */
+        public Builder renameClass(String oldName, String newName) {
+            return renameClassTag(oldName, newName)
+                .xmlChangeAttribute("//@class[.='" + oldName + "']", newName)
+                .renameType(oldName, newName);
+                //TODO update reference attributes
```
Comment (aledsage) [inline core/src/main/java/brooklyn/entity/rebind/transformer/CompoundTransformer.java]: Not sure what this TODO means?

**Comment by aledsage** [comment]
Looks good; merging.

File: core/src/main/java/brooklyn/entity/rebind/transformer/CompoundTransformer.java (hunk 1 above)
Comment (neykov) [inline core/src/main/java/brooklyn/entity/rebind/transformer/CompoundTransformer.java]: When the same object is used at multiple places it is serialized only once and referenced from all other places through the `reference` attribute along with a relative path to the node. When renaming classes the path changes so the value needs to be updated. Not needed for the advanced-networking transforms so I didn't spend time implementing it.

**Pull request merged**